agent/src/main.cpp

```diff
@@ -105,17 +118,24 @@ int main(int argc, char **argv)
     sessionBusOpt.set_short_name('b');
     sessionBusOpt.set_description("Use the session bus instead of the system bus");
 
-    int preloadCount = 0;
+
+    /* Default values need to be somehting that should not be set explicitly
+     * by the user.
+     */
+    Glib::ustring configPath = "";
+    int preloadCount = -1;
     int userID = 0;
-    bool shutdownContainers = true;
-    int timeout = 1;
+    bool keepContainersAlive = false;
+    int timeout = -1;
```

Comment:
No it's not. But what would be a sensible default? Some of the configs I think will require us to do sanity checks after the priority and logic of choosing source is done. I.e. this default should only be used to check if the user did something explicit but never propagate downwards.